wd_helper.au3

```diff
@@ -285,29 +285,31 @@ EndFunc
 ;                                         $_WD_OPTION_Visible (1) = Confirm element is visible
 ;                                         $_WD_OPTION_Enabled (2) = Confirm element is enabled
 ;                                         $_WD_OPTION_Element (4) = Return found element
+;                                         $_WD_OPTION_NoMatch (8) = Confirm element not found
 ;
 ; Return values .: Success      - 1 or element ID
 ;                  Failure      - 0 and sets the @error flag to non-zero
 ;                  @error       - $_WD_ERROR_Success
 ;                  				- $_WD_ERROR_Timeout
 ; Author ........: Dan Pollak
-; Modified ......:
+; Modified ......: mLipok
 ; Remarks .......:
 ; Related .......:
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_WaitElement($sSession, $sStrategy, $sSelector, $iDelay = Default, $iTimeout = Default, $iOptions = Default)
 	Local Const $sFuncName = "_WD_WaitElement"
-	Local $iErr, $iResult = 0, $sElement, $lIsVisible = True, $lIsEnabled = True
+	Local $iErr, $iResult = 0, $sElement, $bIsVisible = True, $bIsEnabled = True
 
 	If $iDelay = Default Then $iDelay = 0
 	If $iTimeout = Default Then $iTimeout = $_WD_DefaultTimeout
 	If $iOptions = Default Then $iOptions = $_WD_OPTION_None
 
-	Local $lVisible = BitAND($iOptions, $_WD_OPTION_Visible)
-	Local $lEnabled = BitAND($iOptions, $_WD_OPTION_Enabled)
-	Local $lReturnElement = BitAND($iOptions, $_WD_OPTION_Element)
+	Local $bVisible = BitAND($iOptions, $_WD_OPTION_Visible)
+	Local $bEnabled = BitAND($iOptions, $_WD_OPTION_Enabled)
+	Local $bReturnElement = BitAND($iOptions, $_WD_OPTION_Element)
+	Local $b_Check_NoMatch = BitAND($iOptions, $_WD_OPTION_NoMatch)

```

Comment:
Probably should add some checks here since $_WD_OPTION_NoMatch can't occur with the other options

Reply:
> $_WD_OPTION_NoMatch can't occur with the other options

not sure what you mean

Reply:
It seems to me that $_WD_OPTION_NoMatch is mutually exclusive from the other options. IE --How can you check for the non-existence of an element *and* then test it's visibility, enabled status, etc.

Reply:
> How can you check for the non-existence of an element and then test it's visibility, enabled status

Absolutely I do not want to check other options.
Just wait till not exist .... I mean waiting to disapear.

Reply:
Correct... If the $_WD_OPTION_NoMatch is specified, ensure that all of the others were not specified. Otherwise, return an error to indicate invalid option.

Reply:
Will do this next few days